shell.nix

```diff
@@ -8,7 +8,7 @@ with import
 
 let
   # unstable packages
-  electron = electron_30; # use the same version as defined in packages/suite-desktop/package.json
+  electron = electron; # use the same version as defined in packages/suite-desktop/package.json. Note that the 31 version is named just "electron"
```

Comment:
Not entirely sure if we want to use it, if it doesn't have specific version... Could it break in future?

Reply:
i think it breaks desktop builds on nixos, but 31.4 [should be there](https://github.com/NixOS/nixpkgs/commit/98636237874295db2a5b8b46d784e74a783d74c5) already

Reply:
Well, there is `electron-chromedriver_31` listed, but not `electron_31`. [They did add it](https://search.nixos.org/packages?channel=24.05&show=electron&from=0&size=50&sort=relevance&type=packages&query=electron), but only named as `electron`. At first I thought it's no problem, but you're right, it will break desktop builds when they bump to 32 at nixos repository.

So I'm gonna drop the electron bump from this PR and finish that later. I'll write an email to the maintainers to ask about it :slightly_smiling_face: